### PR TITLE
Force bump for renovate to catch up

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

File: Makefile [hunk 1]
```diff
@@ -348,6 +348,15 @@ gowork: ## Generate go.work file to support our multi module repository
 	go work use ./api
 	go work sync
 
+PHONY: force-bump
+force-bump: ## Force bump after tagging
+	for dep in $$(cat go.mod | grep  openstack-k8s-operators | grep -vE -- 'indirect|dataplane-operator' | awk '{print $$1}'); do \
```
Comment (jpodivin) [inline Makefile:353]: You can use regular expressions in `awk` to make this a bit more concise.  Same goes for the second script.

**Review by jpodivin — CHANGES_REQUESTED** [review]
If this is supposed to be triggered manually, I would suggest writing some notes about intended usage.

File: Makefile (hunk 1 above)
Comment (fao89) [reply Makefile:353]: I prefer readability to concision

**Review by slagle — APPROVED** [review]

**Pull request merged**